lib/rubocop/runner.rb

```diff
@@ -72,6 +80,8 @@ def inspect_files(files)
     end
 
     def each_inspected_file(files)
+      trap_interrupt
```

Comment:
Does this need to be done for each file?

Reply:
I don't think so, we should install the interrupt handling just once before analysis starts. In any case, the handler is completely going away in #6461.